packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2751,7 +2753,7 @@
   "The_message_is_a_discussion_you_will_not_be_able_to_recover": "The message is a discussion you will not be able to recover the messages!",
   "The_peer__peer__does_not_exist": "The peer <em>__peer__</em> does not exist.",
   "The_redirectUri_is_required": "The redirectUri is required",
-  "The_server_will_restart_in_s_seconds": "The server will restart in %s seconds",
+  "The server will restart in %s seconds": "The server will restart in %s seconds",
```

Comment:
I think you have damaged the translation key here

Reply:
Oh, damn. Sorry about that, fixed.